source/opt/basic_block.cpp

```diff
@@ -101,13 +109,31 @@ void BasicBlock::ForMergeAndContinueLabel(
   --ii;
   if (ii == insts_.begin()) return;
   --ii;
-  if (ii->opcode() == SpvOpSelectionMerge || 
-      ii->opcode() == SpvOpLoopMerge)
-    ii->ForEachInId([&f](const uint32_t* idp) {
-      f(*idp);
-    });
+  if (ii->opcode() == SpvOpSelectionMerge || ii->opcode() == SpvOpLoopMerge)
+    ii->ForEachInId([&f](const uint32_t* idp) { f(*idp); });
+}
+
+uint32_t BasicBlock::MergeBlockIdIfAny(uint32_t* cbid) {
+  auto merge_ii = cend();
+  --merge_ii;
+  if (cbid != nullptr) {
+    *cbid = 0;
+  }
+  uint32_t mbid = 0;
```

Comment:
This code looks a lot like trying to fine the merge instruction.  Could we call "GetMergeInst" instead?